package.json

```diff
@@ -186,7 +187,8 @@
     "typescript": "5.6.2",
     "typescript-eslint": "^8.13.0",
     "wait-port": "^0.3.0",
-    "yargs": "^17.7.2"
+    "yargs": "^17.7.2",
+    "concurrently": "^8.2.2"
```

Comment:
@djabarovgeorge Can we achieve the same parallel execution using NX, avoiding another dependency?

Reply:
I was not able to make it work with NX.

Reply:
It should be `pnpm nx run-many --targets=start --projects=@novu/api-service,@novu/worker,@novu/dashboard --parallel`.

See https://nx.dev/recipes/running-tasks/run-tasks-in-parallel. I don't think NX has the ability to set color per task yet thought but that's not too critical.

Reply:
Yeah, but the color makes it easier to differentiate them, which helps with running all the services in a single terminal. I'll remove it for now since it's not a priority at the moment.